### PR TITLE
Fix UnboundLocalError in FSMinSize (#1249304)

### PR DESCRIPTION
**Pull request by vojtechtrefny**

### CODE REVIEW

File: blivet/tasks/fsminsize.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def _getResizeInfo(self):
         except OSError as e:
             error_msg = "failed to gather info from resize program: %s" % e
         if rc:
-            error_msg = "failed to gather info from resize program: %s" % e
+            error_msg = "failed to gather info from resize program: %s" % rc
```
Comment (vpodzime) [inline blivet/tasks/fsminsize.py]: This should probably rather use ``%d``, but it's not a big deal.

**Comment by vpodzime** [comment]
Looks good to me either way.

File: blivet/tasks/fsminsize.py [hunk 2]
```diff
@@ -54,7 +54,7 @@ def _getResizeInfo(self):
         except OSError as e:
             error_msg = "failed to gather info from resize program: %s" % e
         if rc:
-            error_msg = "failed to gather info from resize program: %s" % e
+            error_msg = "failed to gather info from resize program: %d" % rc
```
Comment (bcl) [inline blivet/tasks/fsminsize.py]: Seems like this would be better inside the try: just so it's clear it only runs when it succeeds.

Comment (vojtechtrefny) [inline blivet/tasks/fsminsize.py]: Makes sense. Fixed.

**Comment by bcl** [comment]
Looks good to me.

**Comment by vojtechtrefny** [comment]
Pushed.

**Pull request closed without merging**